src/lofreq/snpcaller.c

```diff
@@ -1100,6 +1102,14 @@ snpcaller(long double *snp_pvalues,
         goto free_and_exit;
     }
 
+    double mu = 0;
+    for (int i = 0; i < num_err_probs; ++i) {
+        mu += err_probs[i];
+    }
+    const long double poibin_approximation = 1 - gsl_cdf_poisson_P(max_noncons_count - 1, mu);
```

Comment:
There is a `gsl_cdf_poisson_Q` for calculating the survival function of the Poisson that we may use instead.